src/DevSoc.Eventathon/Models/EventResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DevSoc.Eventathon.Calendars.Models;
+
+public class EventResponse
+{
+    public int id { get; set; }
```

Comment:
Uppercase pls

Reply:
I'll change these later when the type definitions and fields get updated because these almost certainly don't match what we want and what we can get from CalDev.